forui/lib/src/widgets/button/button_content.dart

```diff
@@ -37,6 +37,59 @@ final class _FButtonContent extends StatelessWidget {
   }
 }
 
+final class _FButtonIconContent extends StatelessWidget {
+  final Widget child;
+
+  const _FButtonIconContent({required this.child});
+
+  @override
+  Widget build(BuildContext context) {
+    final (:style, enabled: _) = FButton._of(context);
+
+    return Padding(
+      padding: style.iconContent.padding,
+      child: child,
+    );
+  }
+}
+
+/// [FButton] icon content's style.
+class FButtonIconContentStyle with Diagnosticable {
+  /// The padding.
+  final EdgeInsets padding;
+
+  /// Creates a [FButtonIconContentStyle].
+  FButtonIconContentStyle({
```

Comment:
Can this constructor be constant? If so, we can provide `FCustomButtonStyle.iconContent` with a default value in its constructor.

Reply:
Done